tests/manager/addCustomer/managerCanAddNewCustomer.spec.js

```diff
@@ -25,4 +26,22 @@ usage:
 
  2. Do not rely on the customer row id for the steps 8-11. Use the ".last()" locator to get the last row.
 */
+
+const addCustomerPage = new AddCustomerPage(page); 
+const firstName = faker.person.firstName();
+const lastName = faker.person.lastName();
+const postCode = faker.location.zipCode(); 
+
+await addCustomerPage.open();
+await addCustomerPage.fillFirstNameInputField(firstName);
+await addCustomerPage.fillLastNameInputField(lastName);
+await addCustomerPage.fillPostCodeInputField(postCode);
+await addCustomerPage.clickAddCustomerButton();
+await addCustomerPage.reload();
+
+await addCustomerPage.clickCustomersButton();
+await addCustomerPage.assertCustomersTableContainsFirstName(firstName);
+await addCustomerPage.assertCustomersTableContainsLastName(lastName);
+await addCustomerPage.assertCustomersTableContainsPostalCode(postCode);
```

Comment:
The assertions for checking the first name, last name, and postal code rely on the last row being the newly added customer. This might not be reliable if the table is sorted or updated dynamically. Consider implementing a more robust way to verify the customer details.